### PR TITLE
Inputmessage

### PR DESCRIPTION
**Pull request by vistajess**
docs

Addresses https://github.com/dev-onenetworkecommerce/aia/issues/129

### CODE REVIEW

File: docs/sections/Docs-Forms/index.jsx [hunk 1]
```diff
@@ -72,13 +72,21 @@ export default class Forms extends React.Component {
 
         <section className="doc-bottom-space-large">
           <h3 className="doc-heading">Form Error</h3>
-          <p>To apply, .message</p>
+          <p>To apply to bottom, .message</p>
```
Comment (JefMari) [inline docs/sections/Docs-Forms/index.jsx]: Indicate it as default rather 
or like this
`To apply, .message (position in bottom by default)`
or
not indicate the `bottom` anymore

**Comment by JefMari** [comment]
After this is merged, ill work on improving its documentation :+1:

File: docs/sections/Docs-Forms/index.jsx (hunk 1 above)
Comment (srph) [inline docs/sections/Docs-Forms/index.jsx]: No need for this message.

File: docs/sections/Docs-Forms/index.jsx [hunk 2]
```diff
@@ -72,13 +72,21 @@ export default class Forms extends React.Component {
...
           <section className="doc-bottom-space">
             <div className="form-group">
               <input className="form-control -crimson" type="text" placeholder="Enter something.." />
               <div className="message">Error invalid something..</div>
             </div>
           </section>
+
+          <p>To apply message to top , .message .-top</p>
```
Comment (srph) [inline docs/sections/Docs-Forms/index.jsx]: > Places the message above, useful for when an overlapping element is expected (such as date picker).

**Comment by srph** [comment]
LOL, you didn't commit `form.scss`.

**Comment by srph** [comment]
Please use descriptive titles and maintain a clean commit history next time. Thanks.

**Pull request merged**